docsets/Pillow/docset.json

```diff
@@ -11,6 +11,10 @@
         {
             "version": "2.8.2",
             "archive": "versions/2.8.2/Pillow.tgz",
+        },
+        {
+            "version": "3.3.0",
```

Comment:
Newer versions need to appear before older ones (i.e. `3.3.0` should be before `2.8.2`).